app/serializers/recipe_serializer_show.rb

```diff
@@ -0,0 +1,3 @@
+class RecipeSerializerShow < ActiveModel::Serializer
+  attributes :id, :name, :content, :cooking_time
+end
```

Comment:
You can inherit that serializer from `RecipeSerializer` and just add what is missing.